descriptors/ModuleDescriptor-template.json

```diff
@@ -44,6 +44,16 @@
           "pathPattern": "/users/profile-picture",
           "permissionsRequired": [ "users.profile-picture.item.post" ]
         },
+        {
+          "methods": [ "PUT" ],
+          "pathPattern": "/users/configurations/entry/{configId}",
```

Comment:
Do we need entry in path ?

Reply:
Yes , its needed . RMB is unable to generate methods for this API 
users/configurations/{configId}.